leftwm-core/src/handlers/window_handler.rs

```diff
@@ -40,18 +40,18 @@ impl<C: Config, SERVER: DisplayServer> Manager<C, SERVER> {
             && self.state.focus_manager.behaviour.is_sloppy()
             && self.state.focus_manager.sloppy_mouse_follows_focus
             && on_same_tag;
-        //let the DS know we are managing this window
+        // Let the DS know we are managing this window.
         let act = DisplayAction::AddedWindow(window.handle, window.floating(), follow_mouse);
         self.state.actions.push_back(act);
 
-        //let the DS know the correct desktop to find this window
+        // Let the DS know the correct desktop to find this window.
         if !window.tags.is_empty() {
             let act = DisplayAction::SetWindowTags(window.handle, window.tags);
             self.state.actions.push_back(act);
         }
 
         // Tell the WM the new display order of the windows.
-        self.state.sort_windows();
+        self.state.sort_windows(); // Is this needed??
```

Comment:
Is this sort needed, everything should be in an order which works relative to the visible windows.